src/create.lua

```diff
@@ -80,6 +96,10 @@ local function create(className, props)
 		end
 	end
 
+	if table.find(GUI_BASE_2D, className) then
```

Comment:
Can you not use instance:IsA?

Reply:
That would have a significant reflection overhead.